.github/workflows/positron-windows-nightly.yml

```
@@ -104,8 +96,24 @@ jobs:
           POSITRON_R_VER_SEL: 4.4.0
         id: electron-smoke-tests
         run: |
+          Write-Output $env:BUILD_ARTIFACTSTAGINGDIRECTORY # debug
           cd test\smoke
           node run-tests.js --tracing --win
+          Get-ChildItem -Recurse -Filter "*results.xml" # debug
+
+      - name: Set TestRail Run Title
+        id: set-testrail-run-title
+        if: always()
```

```suggestion
        if: success() || failure()
```

By using `if: success() || failure()` instead of `always()`, the step will run only if the previous steps were executed and completed with either a success or failure status. This ensures that it won’t run if the job was cancelled or skipped entirely.